data_service/src/match_processing/match_parser.rs

```diff
@@ -0,0 +1,98 @@
+use riven::models::match_v5::Match;
+
+use crate::types::PuuidToChampionMapping;
+
+pub async fn parse_match(match_data: Match) -> Vec<PuuidToChampionMapping> {
+    // Get match data.
+
+    let to_return: Vec<PuuidToChampionMapping> = match_data
```

Comment:
nit: could leave out both the semicolon and the assignment ehre and just return

Reply:
resolved, please check